index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Youtube Replica</title>
+</head>
+<body>
+    <div class="nav"><!--flex container-->
+        <div class="logo items"><img src="./images/YouTube_Logo.png" alt="logo" class ='logo'> </div>
+        
```

Comment:
I think it will be better to close all HTML elements with div tag

Reply:
- [ ] [optional] It would be nice adding why they need to close all HTML elements, although it might seem obvious to some as long as the code is working the wont understand.